### PR TITLE
[INJIMOB-1303] kotlin library for inji-vci-client

### PR DESCRIPTION
**Pull request by KiruthikaJeyashankar**

### CODE REVIEW

File: kotlin/readme.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Installation
+
+TODO: add installation guide
+
+## APIs
+
+##### Request Credential
+
+Initiates a credential request to the credential issuer and returns back the credential.
```
Comment (swatigoel) [inline kotlin/readme.md]: Request for credential from the providers (credential issuer), and receive the credential back.

File: kotlin/readme.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+|--------------|------------------------------------|--------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| issuerMeta   | IssuerMeta                         | Data object of the issuer details                                                                            | `IssuerMeta(credentialAudience, credentialEndpoint, downloadTimeout, credentialType, credentialFormat)` |
+| signer       | Function  (ByteArray) -> ByteArray | Function which is called to get the signature passing the preHash (Base64EncodedHeader.Base64EncodedPayload) | `fun signer(preHash: ByteArray): ByteArray {//Signing logic}`                                           |
+| accessToken  | String                             | Token provided by the credential issuer post authorization and token request                                 |                                                                                                         |
```
Comment (swatigoel) [inline kotlin/readme.md]: Description -> token issued by providers based on auth code

File: kotlin/vci-client/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        }
+    }
+}
+
+dependencies {
+    implementation("androidx.core:core-ktx:1.9.0")
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("com.google.android.material:material:1.11.0")
+    implementation("com.squareup.okhttp3:okhttp:4.12.0")
+    //TODO: Remove unused packags
```
Comment (swatigoel) [inline kotlin/vci-client/build.gradle.kts]: what is pending here?

File: kotlin/vci-client/src/main/java/io/mosip/vciclient/dto/IssuerMeta.kt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package io.mosip.vciclient.dto
+
+import io.mosip.vciclient.constants.CredentialFormat
+
+class IssuerMeta(
```
Comment (swatigoel) [inline kotlin/vci-client/src/main/java/io/mosip/vciclient/dto/IssuerMeta.kt]: Rename to IssuerMetaData

**Review by swatigoel — CHANGES_REQUESTED** [review]

**Comment by swatigoel** [comment]
Please send app id as traceabilityId and use for logging
Refer this https://docs.mosip.io/inji/inji-mobile-wallet/specifications/face-sdk-specifications

File: kotlin/vci-client/build.gradle.kts (hunk 3 above)
Comment (KiruthikaJeyashankar) [reply kotlin/vci-client/build.gradle.kts]: Missed to remove TODO will remove it

**Review by swatigoel — APPROVED** [review]

**Pull request merged**